docs/docs/tools/Shopify Product Search, Recommendations and Comparison.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 2
+---
+
+Combines search engine, recommendations engine and comparison engine into a cohesive experience of a smart shopping assistant. It will help your customers to make informed decisions faster by helping them discover what products are available, recommend products based on their needs, enquire about product details and compare them.
+
+## Functionality
+
+### Recommendations
+
+Sometimes you don’t know what exactly you want. Our chatbot can recommend products based on user needs, using data available in your store.
+
+### Search
+
+Once you know what kind of product you’re after, you can search for products from your store. Simply ask the chatbot whether product is available.
+
+You can leverage filters to get exactly what you’re looking for and discover what types of products are available.
+
+To mention: price range, sorting, filters, two types of outputs -- summary and details.
+
+### Details and comparison
+
+You can enquire about product details included in descriptions and metadata, including custom metafields.
+
+- Use case: back in stock date. Per-variant back in stock is coming soon.
+
+You can compare 1-3 products in detail and find out what are the main characteristics of larger number of items.
+
+## Setup
+
+### Install Algomo app in your Shopify Store
+
+1. [https://apps.shopify.com/algomo](https://apps.shopify.com/algomo)
+2. …
+
+### Create a tool
+
+1. Create a tool
+
+### Activate the tool
+
+1. Assign tool to a chatbot
+2. Assign chatbot to a widget
+3. Test the widget
+
```

Comment:
I think a section here about how to test the tool would be great

Test the tool
you can ... by ...
For example... "I'm looking for a jumper"